### ISSUE TITLE
Uninstalling kernel-lts2018 bundle while the LTS kernel is active prevents keyboard & ssh from working on next boot.

### ISSUE THREAD

**Issue by eadamsintel** [clearlinux/distribution#1003]
I first noticed this with release 30300 and have reproduced this on two systems.

*Steps to Reproduce
```
sudo swupd bundle-add kernel-lts2018
sudo clr-boot-manager set-kernel org.clearlinux.lts2018.4.19.28-10
sudo reboot now
```
After reboot
```
uname -r
sudo swupd bundle-remove kernel-lts2018
sudo reboot now
```
*What to expect
System should reboot to the latest Clear Linux kernel and be functional

*What happens
System reboots but the USB keyboard doesn't work and ssh doesn't work. I can't figure out how to repair it.

**Comment by bryteise** [comment]
Did you run clr-boot-manager update after the set-kernel command?

**Comment by eadamsintel** [comment]
No. I never had to before.  Just running set-kernel and then rebooting changed the kernel when I did it.

**Comment by eadamsintel** [comment]
I have an interesting update.  I was using a Logitech K400 combo usb keyboard/mouse which was not working. When I switched to just a keyboard I could log into my system. I had no IP address and could not do a clr-boot-manager update.  However, I was able to list the 5.0 kernel and set it. Rebooting fixed my usb keyboard/mouse issue and networking issue once I set it to the correct kernel. Regardless, if you swupd remove a lts kernel bundle it should set the kernel back as part of that process so the system is not left in a bad state.

**Comment by lebensterben** [comment]
I had removed the native kernel bundle and installed LTS kernel bundle. Then I manually set default kernel in systemd-boot menu to one of the LTS kernel.

Later I removed LTS kernel bundle and added back native kernel bundle and set default kernel to the native one in clr-boot-manager. But it turns out that I need to unset default kernel in systemd-boot menu otherwise the selection in clr-boot-manager won't be effective.

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by eadamsintel** [comment]
As an FYI if I run clr-boot-manager update it always reverts back to the latest kernel.

```
$ uname -r
5.1.16-795.native
$ sudo clr-boot-manager list-kernels
* org.clearlinux.native.5.1.16-795
  org.clearlinux.lts2018.4.19.58-61
$ sudo clr-boot-manager set-kernel org.clearlinux.lts2018.4.19.58-61
$ sudo clr-boot-manager list-kernels
  org.clearlinux.native.5.1.16-795
* org.clearlinux.lts2018.4.19.58-61
$ sudo clr-boot-manager update
$ sudo clr-boot-manager list-kernels
* org.clearlinux.native.5.1.16-795
  org.clearlinux.lts2018.4.19.58-61
$ sudo reboot now
Connection to 192.168.25.211 closed by remote host.
```
After reboot
```
$ uname -r
5.1.16-795.native
```

Perhaps there are two issues going on. I wouldn't intuitively think that clr-boot-manager update would revert to the latest kernel.
